doc/installation.rst

```diff
@@ -0,0 +1,51 @@
+Installation
+============
+
+There are two ways to install `tensorwaves`. Even though the PyPI installation
+is the most convenient, we recommend following the :ref:`interactive
+installation <installation:Interactive installation>` procedure.
+
+
+Installation through PyPI
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`tensorwaves` is `available as a PyPI package <https://pypi.org/>`_ so
+installation is super easy:
+
+.. code-block:: shell
+
+  pip install tensorwaves
```

Comment:
Make a comment that right now its under dev and no package exists yet

Reply:
https://github.com/ComPWA/tensorwaves/pull/27/commits/a619a3a80dbecff93e7e009c2070fb9016e27ad9